packages/fuel-indexer-macros/src/indexer.rs

```diff
@@ -478,7 +494,7 @@ fn process_fn_items(
             pub fn decode_logdata(&mut self, rb: u64, data: Vec<u8>) {
                 match rb {
                     #(#logged_types),*
-                    _ => panic!("Unknown logged type id '{}'.", rb),
+                    _ => Logger::warn("Unknown logged type ID; check ABI to make sure that logged types are well-formed")
```

Comment:
- So we still want to keep these compile time panics
- If the user is trying to decode unrecognized logged types that's a sign of a bigger issue -- you don't wanna fail silently there